hail/python/hail/vds/combiner/variant_dataset_combiner.py

```diff
@@ -280,7 +279,7 @@ def new_combiner(*,
                  gvcf_paths: Optional[List[str]] = None,
                  vds_paths: Optional[List[str]] = None,
                  vds_sample_counts: Optional[List[int]] = None,
-                 intervals: Optional[List[hl.utils.Interval]] = None,
+                 intervals: Optional[Union[List[Interval], List[Struct]]] = None,
```

Comment:
The other arg type isn't a list of struct, it's list[interval[Locus]] vs List[Interval[Struct]]. Just back these changes out, we can add them separately but not urgent.